src/osd/PeeringState.cc

```diff
@@ -5427,6 +5426,7 @@ PeeringState::Backfilling::react(const DeferBackfill &c)
     return transit<NotBackfilling>();
   } else {
     // raced with MOSDPGBackfill::OP_BACKFILL_FINISH, ignore
+    psdout(10) << "defer backfill, event discarded: PG does not need backfill" << c.delay << dendl;
```

Comment:
nit: Backfill is not deferred here, we shouldn't print "defer backfill". Moreover, there seem to be no reason to print `c.delay` as it won't be used. How about:
```
    psdout(10) << "discarding stale DeferBackfill event , pg does not need backfill anymore" << c << dendl;
```